### PR TITLE
The news Week clone

### PR DESCRIPTION
**Pull request by Ralph-1**
This is a clone of the news week project showing the use of

- Bootstrap

- html

- css

- font awesome icons

-

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @Ralph-1 . Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] Missing README
- [ ] Linter is not passing all checks. Fix all Stylelint Issues.
- [ ] Erase all commented rules in the CSS file
- [ ] Use semantic HTML for the `<footer>` and each one of the `<article>`s
- [ ] The project is unfinished.
  - [ ] Missing desktop view no element is responsive for this size
  - [ ] Missing Tablet view some elements
  - [ ] Missing mobile view some elements
  - [ ] Missing sections
    - [ ] It contains `Latest news` section ([link to page](https://www.newsweek.com/))
    - [ ] It contains `In the magazine` section ([link to page](https://www.newsweek.com/))
    - [ ] It contains `Editor's picks` section ([link to page](https://www.newsweek.com/))
image
- [ ] Element is not looking as expected on mobile view
image
- Navbar is not looking as original on mobile view, tablet and desktop view
image
- The debate, and the opinion section is not looking the same on all views and doesn't look like the original
image
image
- Article titles jump from one format to another depending on the view
image
image

### [Comments] :green_circle: 

- Use the media query inspector to observe the changes in the original and try to mimic style and responsiveness on the clone.

- Please submit until you have finished the project.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json/
```
Comment (descholar-ceo) [inline .gitignore:2]: Remove the package-lock.json from .gitignore file

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @Ralph-1  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] The debate section, there is an horizontal bar which is passing behind the `VS` dark button

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-11 15-37-54|Screenshot from 2020-12-11 15-38-37 |

:point_right: Could you see the difference :question: :question: :question: 


- [ ] Fix the navbar and the footer on mobile screens

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-11 15-39-39| Screenshot from 2020-12-11 15-40-30|

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Implement all changes I suggested in inline comments in your codes
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by mimipeshy** [comment]
## STATUS : CHANGES REQUIRED 🔴 ♻️ 

Hello @Ralph-1  👋 👋 
✔️ Good job on working so hard on this project, it looks awesome just like the original page
✔️ Good job on working on the TSE's reviews

You are almost done with the project, just fix one more thing as i have noted below ⬇️ 


- [ ] As pointed out by the previous TSE,  Fix the footer on mobile screens. The articles are repeating themselves after the footer articles.
Footer clone
image

That's all. 😄 , kindy submit another review when you are done

Regards
Peris Ndanu
Feel free to reach out on slack if you have any questions

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:



_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request merged**